eslint-config-base/index.js

```diff
@@ -138,6 +269,59 @@ module.exports = {
         'ignoreRegExpLiterals': true
       }
     ],
+    'no-multi-spaces': [2, {
+      'ignoreEOLComments': false
+    }],
+    'no-multi-str': 2,
+    'function-call-argument-newline': [2, 'consistent'],
+    'func-style': [2, 'expression'],
+    'function-paren-newline': [2, 'multiline-arguments'],
+    'newline-per-chained-call': [2, { ignoreChainWithDepth: 4 }],
+    'no-mixed-operators': [2, {
+      // the list of arithmetic groups disallows mixing `%` and `**`
+      // with other arithmetic operators.
+      groups: [
+        ['%', '**'],
+        ['%', '+'],
+        ['%', '-'],
+        ['%', '*'],
+        ['%', '/'],
+        ['/', '*'],
+        ['&', '|', '<<', '>>', '>>>'],
+        ['==', '!=', '===', '!=='],
+        ['&&', '||']
+      ],
+      allowSamePrecedence: false
+    }],
+    'no-tabs': 2,
+    'no-whitespace-before-property': 2,
+    'nonblock-statement-body-position': [2, 'any'],
+    'object-curly-newline': [2, {
+      ObjectExpression: { minProperties: 4, multiline: true, consistent: true },
+      ObjectPattern: { minProperties: 4, multiline: true, consistent: true},
+      ImportDeclaration: { minProperties: 4, multiline: true, consistent: true},
+      ExportDeclaration: { minProperties: 4, multiline: true, consistent: true }
+    }],
+    'object-property-newline': [2, { allowAllPropertiesOnSameLine: true }],
+    'one-var': [2, 'never'],
+    'one-var-declaration-per-line': [2, 'always'],
+    'newline-after-var': [2, 'always'],
+    'operator-assignment': [2, 'always'],
+    'padding-line-between-statements': [2, { blankLine: 'always', prev: '*', next: 'return' }],
+    'padded-blocks': [2, {
+      blocks: 'never',
+      classes: 'never',
+      switches: 'never'
+    }, {
+      allowSingleLineBlocks: true
+    }],
+    'quote-props': [2, 'as-needed', { keywords: false, unnecessary: true, numbers: false }],
+    'spaced-comment': [2, 'always', {
```

Comment:
We got errors because of this rule in comments like 
```javascript
//////////
// Report options
//////////
const reportOptions = {
  sheetName: 'Converted report',
  fontSizeSmall: 8,
  fontSizeTableSubHeadings: 10,
  fontSizeTitle: 20,
  fontSizeSubTitle: 14,
  currencyFormat: '0.00',
  dateFormat: 'dd mmm yyy',
};
//////////
```

We use such an approach for Google Apps Script when the code might be opened with the build-in script editor.

Can we somehow make errors only appear when we have a letter after the `//` without a space?

Reply:
ok, I will add rule for that case